jina/executors/indexers/keyvalue.py

```diff
@@ -52,15 +52,19 @@ def __init__(self, *args, **kwargs):
         self._start = 0
         self._page_size = mmap.ALLOCATIONGRANULARITY
 
-    def add(self, docs: Iterator['jina_pb2.Document'], *args, **kwargs):
-        for d in docs:
-            s = d.SerializeToString()
-            l = len(s)  #: the length
+    def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
```

Comment:
Keys is an iterator of string since change in doc id

Reply:
Since we are passing `id2hash` as key, that would be an iterator of int

Reply:
Good point, just ignore it then